### PR TITLE
Updated cap command to include CS , CU

### PR DESCRIPTION
**Pull request by tobiasceg**

### CODE REVIEW

File: src/main/java/seedu/duke/command/cap/CapCommand.java [hunk 1]
```diff
@@ -92,24 +95,38 @@ private double gradeConvert(String grade) {
...
+            } else {
+                numberOfCsCuModules++;
             }
-            sumMc += ModuleManager.getModule(i).getModuleCredit();
-            mcGrade = ModuleManager.getModule(i).getModuleCredit()
-                    * gradeValue;
-            sumMcGrade += mcGrade;
         }
+        if (numberOfCsCuModules == numberOfModules) {
+            sumMcGrade = 20;
```
Comment (SimJunYou) [inline src/main/java/seedu/duke/command/cap/CapCommand.java]: What's this for?

File: src/main/java/seedu/duke/util/ExceptionMessage.java [hunk 2]
```diff
@@ -51,8 +51,8 @@
             + "Please use 'help' to see our list of commands.\n";
     public static final String MESSAGE_INVALID_GRADE =
             "Your grade input isn't part of the NUS grading scheme. \n"
-            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F.\n"
-            + "Do not enter grades CS and CU - leave the module ungraded!\n";
+            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F, CS, CU. \n"
+            + "Grades have to be in uppercase not lower case. \n";
```
Comment (SimJunYou) [inline src/main/java/seedu/duke/util/ExceptionMessage.java:55]: Possible to turn user input into uppercase automatically?

**Review by SimJunYou — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/command/cap/CapCommand.java (hunk 1 above)
Comment (tobiasceg) [reply src/main/java/seedu/duke/command/cap/CapCommand.java]: checking if that semester they SU all the mods

File: src/main/java/seedu/duke/util/ExceptionMessage.java (hunk 2 above)
Comment (tobiasceg) [reply src/main/java/seedu/duke/util/ExceptionMessage.java:55]: yes boss

**Review by SimJunYou — APPROVED** [review]

**Pull request merged**